doc/rtd/development/integration_tests.rst

```diff
@@ -31,26 +31,179 @@ integration tests. To run all integration tests, you would run:
 
 .. code-block:: bash
 
-    $ tox -e integration-tests
+    tox -e integration-tests
 
-``pytest`` arguments may also be passed. For example:
+``pytest`` arguments may also be passed.
+
+Or, specify a path to run all tests inside the given file or directory:
+
+.. code-block:: bash
+
+    tox -e integration-tests tests/integration_tests/modules/test_combined.py
+
+
+.. code-block:: bash
+
+    tox -e integration-tests tests/integration_tests/modules
+
+Or, run a specific test:
 
 .. code-block:: bash
 
-    $ tox -e integration-tests tests/integration_tests/modules/test_combined.py
+    tox -e integration-tests tests/integration_tests/modules/test_combined.py::test_bootcmd
+
+TODO: Maybe make these 3 commands in one tab group??
+
 
 Configuration
 =============
 
 All possible configuration values are defined in
-`tests/integration_tests/integration_settings.py`_. Defaults can be overridden
+`tests/integration_tests/integration_settings.py`_. Look in this file for
+the full list of variables that are available and for context on what each
+variable does and what the default values are.
```

Comment:
You chopped off the `Defaults can be overridden` text so now the following sentence is a fragment.

Reply:
good catch XD